apps/meteor/ee/tests/unit/apps/meteor/ee/server/local-services/instance/getTransporter.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { expect } from 'chai';
```

Comment:
I think the unit tests should live side by side with the files it calls `colocating`